lightning/src/ln/channel.rs

```diff
@@ -671,6 +671,12 @@ impl<Signer: Sign> Channel<Signer> {
 
 		let feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
 
+		let value_to_self_msat = channel_value_satoshis * 1000 - push_msat;
+		let commitment_tx_fee = feerate as u64 * (COMMITMENT_TX_BASE_WEIGHT + 4 * COMMITMENT_TX_WEIGHT_PER_HTLC) / 1000;
```

Comment:
This should be able to use one of the `commit_tx_fee` helpers

Reply:
There is technically no `self` on which to call it currently. I suppose we could make that function take the fee as an argument instead of `&self`? Would you prefer that?

Reply:
Ah. Yeah, I'd prefer that